test/test_udp_socket.rs

```diff
@@ -1,6 +1,5 @@
 use crate::localhost;
 use bytes::{Buf, MutBuf, RingBuf, SliceBuf};
-use iovec::IoVec;
```

Comment:
This is needed, it fails various build see the Azure pipeline.

Reply:
Ah, that test had `#[cfg(unix)]` which explains why it was showing up as an unused import. I added the `use` in the test function directly to prevent the unused import warning.